tests/unit/python/foglamp/services/core/test_connect.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from unittest.mock import MagicMock, patch
+import pytest
+
+from foglamp.services.core.service_registry.service_registry import ServiceRegistry as Service
```

Comment:
Why use alias? Essentially all imports in unit test should comply with the original code.